sample_app/sampleapp_common.c

```diff
@@ -753,14 +795,22 @@ static int app_state_ind (
             printf ("    No error status available\n");
          }
       }
+
+      /* Set output values */
+      if (p_appdata->arguments.verbosity > 0)
+      {
+         printf ("Setting outputs to default values (due to ABORT)\n");
```

Comment:
Remove if printout is moved to app_set_outputs_default_value()

Reply:
Done